src/internal/helpers/over.ts

```diff
@@ -0,0 +1,13 @@
+import { Operator } from '../Operator';
+import { Subscriber } from '../Subscriber';
+import { Observable } from '../Observable';
+import { OperatorFunction, ObservableInput } from '../types';
+import { map } from '../operators/map';
+import { from } from '../observable/from';
+
+export function over<O, I, R>(
+  selector: (outer: O, index: number) => ObservableInput<I>,
+  project: (outerValue: O, innerValue: I, outerIndex: number, innerIndex: number) => R): (outer: O, index: number) => Observable<R> {
```

Comment:
While I'm still debating whether or not we want to add this directly to the core library or if it should be something standalone, I wanted to point out that the names are flipped as opposed to Rx 5.x... should be `project`, then `selector`.